modules/core/src/main/scala/lucuma/odb/api/repo/ObservationRepo.scala

```diff
@@ -154,5 +159,31 @@ object ObservationRepo {
           Tables.constraintSetObservation,
           ConstraintSetModel.ConstraintSetEvent.updated
         )
+
+      override def unsetConstraintSet(oid: Observation.Id)(implicit F: MonadError[F, Throwable]): F[ObservationModel] = {
+        val focusObsLens = focusOn(oid)
+
+        val obsLens: Lens[Tables, Option[ObservationModel]] =
+          Lens[Tables, Option[ObservationModel]](focusObsLens.get)(oo => focusObsLens.set(oo))
```

Comment:
Does this differ from `focusObsLens`?

Reply:
Nope. I was having troubles getting everything to line up. I was sure I copied these optics from somewhere else and they worked, but I have no idea where. I'm probably imagining things...